flink-runtime/src/test/java/org/apache/flink/runtime/security/SecurityUtilsTest.java

```diff
@@ -31,23 +31,23 @@
 
 import org.apache.flink.shaded.guava31.com.google.common.collect.Lists;
 
-import org.junit.AfterClass;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Test;
 
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
These assert should be replaced with `org.assertj.core.api.Assertions.assertThat`

Reply:
I have replace all asserts with `org.assertj.core.api.Assertions.assertThat/assertThatThrownBy/assertThatCode.`
`assertThatCode` paired with `doesNotThrowAnyException` is used to replace `assertDoesNotThrow`.